### DISCUSSION TITLE
Git status for multiple repos

### DISCUSSION THREAD

**Discussion by AadityaRavindran** [nvim-neo-tree/neo-tree.nvim#1171] [Q&A] [answered]
Is there a way to view git status if the main folder opened has multiple git repos?

**Comment by cseickel** [comment, marked as answer]
No, sorry. The `git_status` source was written with the assumption that you would be working within one repo at a time.

**Reply by AadityaRavindran** [reply]
Thank you for the fast response. Would it be too much of a bother to request for this feature?

**Reply by cseickel** [reply]
It's not something I would personally do because I can't see the use case and it would add a lot of complications. 

I'm not even sure I would accept the PR if someone else wanted to put in the work because it's guaranteed to cause bugs and that's not fair to the majority of people that don't need this feature,
